docs/reference/commandline/create.md

```diff
@@ -80,11 +80,13 @@ Creates a new container.
       --ulimit=[]                   Ulimit options
       --uts=""                      UTS namespace to use
       -v, --volume=[]               Bind mount a volume with: [host-src:]container-dest[:<options>], where
```

Comment:
@rhvgoyal Thank you for the contribution.  Here, the flag description should be short and sweet, a discussion of behavior level for later in the description.  Also, fill the square brackets with the actual format rather than leaving it to the flag statement...so:

```markdown
      -v, --volume=[host-src:]container-dest[:<options>]
                                    Bind mount a volume. The comma-delimited
                                    `options` are [rw|ro], [z|Z], or
                                    [[r]shared|[r]slave|[r]private]. The
                                    'host-src' is an absolute path or a name
                                    value.
```